demos/sample_azure_iot_adu/sample_azure_iot_adu.c

```diff
@@ -181,6 +181,8 @@ static uint32_t ulReportedPropertiesUpdateLength;
 
 uint8_t ucAduContextBuffer[ ADU_CONTEXT_BUFFER_SIZE ];
 
+#define sampleaduDEFAULT_RESULT_DETAILS "Ok"
```

Comment:
nit: we can make this a `uint8_t` buffer.

Reply:
done